### ISSUE TITLE
Y NO UPDATES!?!?!?

### ISSUE THREAD

**Issue by JasonLandbridge** [PlexRipper/PlexRipper#97]
TL;DR; I have been very busy but I am working on the v0.9.0 version (extra 24.000 lines of code so far) => [See here](https://github.com/PlexRipper/PlexRipper/pull/106)

Hi there, 

So for those asking "why no updates", this is what's been happening. I've had a big renovation for 3 weeks but in between I have been continuing to work on version 0.9.0 which I hope to release "soon". Doing several smaller updates to fix many things was not really possible so it will be a big update with many improvements! 

Please note, unfortunately, I have a full-time job so unless someone can donate 1.000.000 in crypto I'm forced to work instead of spending all my time on developing PlexRipper, which I honestly would rather do.

Sneak peak of features:

- Complete rewrite and refactor of the download process, this will fix several major issues
    - Certain downloads being very slow due to a certain bug
    - Downloaded files having root permissions
    - The download progress shown not being accurate
    - Multi-part media downloading
    - A download speed limit configurable per server
  
It's tempting to include more features and fixes in this update but those will be worked on after this. Namely fixing connection and  syncing issues with Plex servers. 

I can guarantee that I have not abandoned PlexRipper and I am still working on it in my spare time!

Keep your feedback coming, I am reading and tracking everything!

**Comment by milfer322** [comment]
Thank you very much for all your work, I look forward to the update! take the time you need, I'm sorry I can't help you because I don't know enough programming!

**Comment by Animizio** [comment]
thanks for this text. i really hope this is not dead in the future and you get enough time for all the love it should get :) good luck my friend
